parsing/g_constr.mlg

```diff
@@ -26,16 +26,6 @@ open Pcoq.Constr
 (* TODO: avoid this redefinition without an extra dep to Notation_ops *)
 let ldots_var = Id.of_string ".."
 
-let constr_kw =
-  [ "forall"; "fun"; "match"; "fix"; "cofix"; "with"; "in"; "for";
-    "end"; "as"; "let"; "if"; "then"; "else"; "return";
-    "SProp"; "Prop"; "Set"; "Type";
-    ":="; "=>"; "->"; ".."; "<:"; "<<:"; ":>";
-    ".("; "()"; "`{"; "`("; "@{"; "{|";
-    "_"; "@"; "+"; "!"; "?"; ";"; ","; ":" ]
-
-let _ = List.iter CLexer.add_keyword constr_kw
```

Comment:
I am not sure about the removal of this static registration. Weren't we doing that as a foolproof guard to ensure that nobody else had the fancy idea to declare some of those as identifiers? Otherwise, what would have been the historical reason for this code?

Reply:
I don't know the historical reason (it seems to come from Coq 6). What I know is that it regularly got desynchronized with the reality of what the grammar was declaring as keywords. These symbols are anyway not parsable as identifiers.

Reply:
@ppedrot Are you OK with Hugo's answer?

Reply:
Yep.